documentation/src/main/asciidoc/topics/ref_query_performance.adoc

```diff
@@ -0,0 +1,39 @@
+[id='query_performance-{context}']
+= Query Performance Tuning
+{brandname} exposes statistics for queries and provides configurable attributes so you can monitor and tune queries for optimal performance.
+
+.Check index usage statistics
+
+Indexing caches improves query performance.
+However, in some cases, queries might only partially use an index such as when not all fields in the schema are annotated.
+
+Start tuning query performance by checking the time it takes for each type of query to run.
+If indexed queries are not faster than non-indexed queries, you should make sure that caches are fully indexed by double checking entities and field mappings.
+
+.Index commit interval
```

Comment:
There are many attributes that affect performance, not only "Index commit interval"

I'd prefer to call the sections "Index perf" and "query perf" like in ```master```.